t/cobrand/bromley.t

```diff
@@ -1833,27 +1815,19 @@ subtest 'Garden Waste new subs alert update emails contain bin collection days l
 
         FixMyStreet::Script::Alerts::send();
 
-        my $alerts = FixMyStreet::App->model('DB::Alert')->search({
-            alert_type => 'new_updates',
-            parameter => $new_sub->id,
-        });
-        is $alerts->count, 1, "alert found for GGW update";
-
-        $mech->email_count_is(1, 'GGW update alert email found');
-        my @emails = $mech->get_email;
-        for my $email (@emails) {
-            my $text_body = $mech->get_text_body_from_email($email);
-            like $text_body, qr/Check your bin collections day/, 'has bin day link text in text part';
-            my @links = $mech->get_link_from_email($email, 'get_all_links');
-            my $found = any { $_ =~ m[recyclingservices\.bromley\.gov\.uk/waste] } @links;
-            ok $found, 'Found bin day URL in text part of alert email';
-
-            my $html_body = $mech->get_html_body_from_email($email);
-            like $html_body, qr/Check your bin collections day/, 'has bin day link text in HTML part';
-            my @uris = $html_body =~ m/$RE{URI}/g;
-            $found = any { $_ =~ m[recyclingservices\.bromley\.gov\.uk/waste] } @uris;
-            ok $found, 'Found bin day URL in HTML part of alert email';
-        }
+        $mech->email_count_is(1, 'Garden waste update alert email found');
```

Comment:
You don't need this line, because the line below performs the same check in `get_email`.